crates/fs/src/fs_watcher.rs

```diff
@@ -89,6 +89,8 @@ pub struct GlobalWatcher {
     pub(super) inotify: Mutex<notify::INotifyWatcher>,
     #[cfg(target_os = "freebsd")]
     pub(super) inotify: Mutex<notify::KqueueWatcher>,
+    #[cfg(target_os = "windows")]
+    pub(super) inotify: Mutex<notify::ReadDirectoryChangesWatcher>,
```

Comment:
inotify is a linux-exclusive API, could we change the name of this variable to be generic?